### PR TITLE
Error with an incorrect trained Haar model

### PR DESCRIPTION
**Pull request by georgedem975**
Fixes https://github.com/opencv/opencv/issues/23497

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [x] There is a [reference](https://github.com/opencv/opencv/issues/23497) to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by georgedem975** [comment]
so far, I have found such a flaw in only one model, but if necessary, I can view all the other models and check them for performance

**Comment by asmorkalov** [comment]
@georgedem975 Could you provide details on the issue? Please submit a ticket with image and sample code or command line to use existing sample to reproduce the issue.

File: data/haarcascades_cuda/haarcascade_fullbody.xml [hunk 1]
```diff
@@ -138,7 +138,7 @@ Thanks to Martin Spengler, ETH Zurich, for providing the demo movie.
 -->
 <opencv_storage>
 <haarcascade_fullbody type_id="opencv-haar-classifier">
-  <size>14 28</size>
+  <size>28 14</size>
```
Comment (opencv-alalek) [inline data/haarcascades_cuda/haarcascade_fullbody.xml:141]: > `14x28 fullbody detector`

That change contradicts the description.

Bodies are usually vertically aligned, so `height > width`

**Review by opencv-alalek — CHANGES_REQUESTED** [review]
I believe this file should be dropped completely (at least for 5.x):
- the file is not tested and not used in OpenCV (however some demo videos are available on YT: e.g, https://www.youtube.com/watch?v=7x1jowSCWNY )
- perhaps file has licensing issues for commercial usage
- provided links on related resources are dead
- this approach is useful for learning/education purpose only. Nowadays DNN-based solutions provides much better accuracy (see "pedestrian detection" task, e.g. from ADAS algorithms)

**Pull request closed without merging**